monkestation/code/modules/art_sci_overrides/artifact_components/_base_component.dm

```diff
@@ -1,13 +1,10 @@
-#define BASE_MAX_ACTIVATORS 2
+#define BASE_MAX_ACTIVATORS 3
+#define BASE_MAX_EFFECTS 3
 
 /datum/component/artifact
 	dupe_mode = COMPONENT_DUPE_UNIQUE
-	///object related to this datum for spawning
-	var/obj/associated_object
-	///actual specific object for this instance
-	var/obj/holder
-	///list weight for picking this artifact datum (0 = never)
-	var/weight = 0
+	//The object we are attached to
+	var/obj/structure/artifact/holder
```

Comment:
this should still be an object unless you are changing the init to disallow base objects from applying to it

Reply:
You right, yea, whoops